wpeverest/user-registration
# UR-852 Enhance - Global Settings Design

### All Submissions:






### Changes proposed in this Pull Request:
This PR implements a new design of the Global Settings as per the [Design in Figma](https://www.figma.com/file/pGx46GyMMoRzTeRme4A6fg/UR-Settings?node-id=2136-35035&t=0RVvVtAgnEB58H7g-0)

Major changes besides design:
1. Switch from Checkbox and Yes/No Select option to Toggle:

### Dependent PRs or branches:
1. WooCommerce - [Master branch](https://github.com/wpeverest/user-registration-woocommerce)
2. Content Restriction - https://github.com/wpeverest/user-registration-content-restriction/pull/25
3. Social Connect - https://github.com/wpeverest/user-registration-social-connect/pull/14
4. Geolocation - [Master branch](https://github.com/wpeverest/user-registration-geolocation)
5. Stripe - https://github.com/wpeverest/user-registration-stripe/pull/23
6. Zapier - [Master branch](https://github.com/wpeverest/user-registration-zapier)
7. Payments - [Master branch](https://github.com/wpeverest/user-registration-payments)
8. Two Factor Authentication - [Master branch](https://github.com/wpeverest/user-registration-two-factor-authentication)
   
### How to test the changes in this Pull Request:

1. Switch to this branch and verify if there are no inconsistencies in the design.
3. Do check the backward compatibilities, and dynamic value update changes for the options with toggle.
4. Verify if settings can be saved.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Global Settings Design
